developer_defaults.tf

```diff
@@ -213,6 +213,11 @@ variable "backup_retention_period" {
   default     = 0
 }
 
+variable "queue_ebs_snapshot_retention_days" {
```

Comment:
This should be with the RabbitMQ variables (after `rabbitmq_delete_volume_on_termination`) rather than the RDS variables